.github/workflows/stateless-e2e.yml

```diff
@@ -0,0 +1,116 @@
+name: Stateless Sync Tests
+
+on:
+  push:
+    branches:
+      - 'stateless_sync'
+  pull_request:
+    branches:
+      - 'stateless_sync'
+    types: [opened, synchronize, reopened]
+
+concurrency:
+  group: stateless-sync-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+env:
+  ENCLAVE_NAME: stateless-sync-e2e
+
+jobs:
+  e2e-tests:
+    name: E2E Tests
+    runs-on: ubuntu-latest
```

Comment:
We can use larger runner (similar to this [workflow](https://github.com/maticnetwork/bor/commit/041ced5e297b86f6d4ec49a5361542f129d35935)), which has 300GB disk space.